features/CLI/CommandLine.java

```diff
@@ -0,0 +1,186 @@
+import java.util.Scanner;
+
+
+public class CommandLine {
+
+	public static void main(String[] args) {
+		Scanner cmdLine = new Scanner(System.in);
+		System.out.println("Hello, and welcome to Team mike's UML editor.");
+		System.out.println("To exit the program, type quit");
+		System.out.println("To see all the commands available, type help");
+
+
+		while(true)
+		{
+			String line = cmdLine.nextLine();
+			//parse command line string into a list of commands by spaces
+			String[] commands = line.split(" ");
+			//get first command
+			String command = commands[0];
+
+			if (command.equals("quit"))
+			{
+				break;
+			}
+			
+			switch(command)
+			{
+				case "help":
+					help(); 
+					break;
+					
+				case "save":
+					//call save depending on if pathname was specified or not
+					if(commands.length == 2)
+					{
+						save(commands[1]);
+					}
+					else if (commands.length == 3)
+					{
+						save(commands[1], commands[2]);							
+					}
+					else
+					{
```

Comment:
```suggestion
					} else {
```
It might be nice to condense these else and else-if statements to take up one line instead of three. Not important but it would keep this switch statement less spread out. Thoughts?

Reply:
I generally prefer the 3 line setup, just because then I can trace down from curly brace to curly brace a little easier as things get more nested further on.

I'll change it in the next commit just to see how everyone likes it